pitop/robotics/drive_controller.py

```diff
@@ -7,7 +7,6 @@
 
 from simple_pid import PID
 
-from pitop.core.exceptions import UninitializedComponent
```

Comment:
@jcapona any reason this hasn't been done to master branch yet? Or is it just something we're going to merge in with this PR? I'm not entirely sure of the implications

Reply:
sorry... yeah, this should be merged with this PR... if you want we can separate it into another one to avoid confussion